pylot/planning/rrt_star/rrt_star_planning_operator.py

```diff
@@ -181,14 +181,25 @@ def _build_obstacle_map(self, vehicle_transform):
             time = 0
             # use all prediction times as potential obstacles
             for location in prediction.trajectory:
+                prediction_transform_ego = pylot.simulation.utils.Transform(
+                    location=pylot.simulation.utils.Location(
+                        x=location.x,
+                        y=location.y,
+                        z=location.z,
+                    ),
+                    rotation=pylot.simulation.utils.Rotation()
+                )
+                # convert to global from ego
+                prediction_transform_global = vehicle_transform * prediction_transform_ego
+                prediction_location_global = prediction_transform_global.location
```

Comment:
Aren't all these lines equivalent to vehicle_transform.transform_points(prediction.trajectory)?

Reply:
Yes :) Forgot this existed, thanks!

Reply:
Done